scripts/run-tests.sh

```diff
@@ -26,6 +27,11 @@ cleanup() {
 
 trap cleanup EXIT
 
+check_is_installed docker
```

Comment:
docker should not be required ?

Reply:
yeah, that is not needed.

we probably should have some enhancements where we can also `build_and_push` code changed images and run test on those (similar to vpc-cni repo)